core/jvm/src/main/scala/me/shadaj/scalapy/interpreter/Platform.scala

```diff
@@ -79,4 +81,9 @@ object Platform {
   def setPtrLong(ptr: Pointer, offset: Int, value: Long): Unit = ptr.setLong(offset, value)
   def setPtrInt(ptr: Pointer, offset: Int, value: Int): Unit = ptr.setInt(offset, value)
   def setPtrByte(ptr: Pointer, offset: Int, value: Byte): Unit = ptr.setByte(offset, value)
+
+  def programName: Option[jna.WString] =
+    Properties.propOrNone("scalapy.python.programname")
+      .orElse(sys.env.get("SCALAPY_PYTHON_PROGRAMNAME"))
```

Comment:
About `sys.env`, I think it's safer to use `System.getenv` instead. A little known fact about `sys.env` is that it's case-sensitive on Windows, while `System.getenv` is case-insensitive (which is the expected behavior there I believe). So using `sys.env` sometimes leads to unexpected and surprising behavior on Windows 😬